src/functions/updatePenaltyGroupWithPayment.unit.js

```diff
@@ -0,0 +1,32 @@
+import sinon from 'sinon';
+
+import updatePenaltyGroupWithPayment from './updatePenaltyGroupWithPayment';
+import PenaltyGroupSvc from '../services/penaltyGroups';
+
+const TEST_ID = 'abc123';
+const PAYMENT_STATUS = 'PAID';
+const PENALTY_TYPE = 'test';
+
+
+describe('updatePenaltyGroupsWithPayment', () => {
+	let penaltyGroupSvc;
+	let callbackSpy;
+	const event = { body: { id: TEST_ID, paymentStatus: PAYMENT_STATUS, penaltyType: PENALTY_TYPE } };
+
+	beforeEach(() => {
+		penaltyGroupSvc = sinon.stub(PenaltyGroupSvc.prototype, 'updatePenaltyGroupWithPayment');
+		penaltyGroupSvc.withArgs(event).resolves();
```

Comment:
Probably worth adding a test case ensuring we do something sensible for a reject too (here and in lots of other places in the project!)

Reply:
Thanks, good point.